packages/webpack-config/package.json

```diff
@@ -31,7 +31,12 @@
     "copy-webpack-plugin": "^5.0.0",
     "css-loader": "^2.1.1",
     "deep-diff": "^1.0.2",
+    "eslint": "^6.0.1",
     "eslint-loader": "^2.2.1",
+    "eslint-plugin-babel": "^5.3.0",
+    "eslint-plugin-flowtype": "^3.12.1",
+    "eslint-plugin-import": "^2.18.1",
+    "eslint-plugin-prettier": "^3.1.0",
```

Comment:
Do we also need eslint-config-universe added here since that's the config we use? usage: https://github.com/expo/expo-cli/pull/860/files#diff-cc24e2c78b9d805246070e6c56fc13a6R451

Killer turnaround time on this >: D

Reply:
Good catch